aksel.nav.no/website/pages/eksempler/alert-with-buttons/alert-with-buttons.tsx

```diff
@@ -0,0 +1,62 @@
+import {
+  Alert,
+  BodyLong,
+  Box,
+  Button,
+  HStack,
+  Heading,
+  VStack,
+} from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <VStack gap="8">
+      <Alert variant="warning">
+        <Heading spacing size="small" level="3">
+          Informasjon om ansvaret ditt
+        </Heading>
+        <BodyLong spacing>
+          Det er viktig at du gjennomfører denne aktiviteten med NAV.
+        </BodyLong>
+        <BodyLong>
+          Gjør du ikke det, kan det medføre at stønaden du mottar fra NAV
+          bortfaller for en periode eller stanses. Hvis du ikke kan gjennomføre
+          aktiviteten, ber vi deg ta kontakt med veilederen din så snart som
+          mulig.
+        </BodyLong>
+        <Box marginBlock="4 0">
```

Comment:
Would it work to instead use `<BodyLong spacing>` rather than `<Box marginBlock="4 0">` here?